### PR TITLE
Created a release script

### PR DESCRIPTION
**Pull request by clepore**
I dunno, what do you think we should do with this @nickwynja ? The only way to fully test it would be when we release next.

A dry run of just comments looks like this:
screen shot 2018-05-14 at 6 29 32 pm

### CODE REVIEW

File: scripts/release.sh [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+fancyEcho "Creating" "git tag for new version"
+git tag v${VERSION_LABEL}
+git push origin tags
+
+fancyEcho "Pushing" "branch '$RELEASE_BRANCH' to github"
+git push origin $RELEASE_BRANCH
+
+fancyEcho "Publishing" "to design.infor.com"
+npx gulp publish
```
Comment (nickwynja) [inline scripts/release.sh]: Specify which `--site` to publish to.

File: scripts/release.sh [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+bump "package.json"
+cd ../../../
+git commit -am "Incrementing version number to $VERSION_LABEL"
+
+fancyEcho "Creating" "git tag for new version"
+git tag v${VERSION_LABEL}
+git push origin tags
+
+fancyEcho "Pushing" "branch '$RELEASE_BRANCH' to github"
+git push origin $RELEASE_BRANCH
```
Comment (nickwynja) [inline scripts/release.sh]: Do we want a new branch `9.9.9` here? Wouldn't we want something like updating `9.9.x`?

Comment (clepore) [reply scripts/release.sh]: Hmm good point. We need the "branch name" and the "version number" to be 2 diff things.

**Comment by clepore** [comment]
Shelving this for now.

**Pull request closed without merging**